### DISCUSSION TITLE
Adding lazy nested components breaks functionality

### DISCUSSION THREAD

**Discussion by andrew1601** [livewire/livewire#6835] [7. Bugs (archived)]
### Livewire version

3.0.4

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/andrew1601/livewire-lazy-bug

### Steps To Reproduce

1. Create a new Laravel project
2. Install livewire
3. Create counter component
4. Create message component that sleeps for a while in `mount()` and provides a placeholder.
5. Nest the message component inside the counter component.
6. Functionality of counter component breaks

### Additional information

I don't know whether this is a bug or not, but if it's not, I can't figure out what I'm doing wrong.

Console output:

<img width="1500" alt="image" src="https://github.com/livewire/livewire/assets/6561258/a0538bbb-e6be-45f6-9a2c-320a212a8a0c">

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by lrljoe** [comment]
You need to wrap the top level view of your component with an empty div (note - not wrapping your component with it!)

This seems to fix the issue where the snapshot can't attach correctly, and thus drops off completely, causing the component id to disconnect

**Reply by yassinetrbls** [reply]
Hello, same issue on my side. Can you provide more explanation for the workaround? Thanks :)

**Reply by lrljoe** [reply]
So your livewire component returns a blade/view in the render.
The first element in this blade should be a div

<div>

Don't add any wire:id/key/attributes etc to it
